tests/test.user-actions.js

```diff
@@ -38,45 +29,47 @@ describe('userActions', function() {
     opt.jqueryOpt.returnedData = opt.data;
 
     var jquery = helpers.fakeJquery(opt.jqueryOpt);
-    userActions.__set__('$', jquery.stub);
 
-    return opt.data;
+    return {
+      data: opt.data,
+      jquery: jquery.stub,
+    };
   }
 
   it('should dispatch sign-in action', function() {
-    setApiSignInResult();
+    var setup = setApiSignInResult();
 
-    userActions.signIn('access-token')(dispatchSpy);
+    userActions.signIn('access-token', setup.jquery)(dispatchSpy);
```

Comment:
I am bummed we can't do `signIn('access-token', jquery=setup.jquery)` but oh well